bin/lib/amazon.py

```diff
@@ -386,3 +391,51 @@ def has_bouncelock_file(cfg: Config):
         return True
     except s3_client.exceptions.NoSuchKey:
         return False
+
+
+def notify_file_name():
+    return 'ce-notify-file'
+
+
+def has_notify_file():
+    try:
+        s3_client.get_object(
```

Comment:
In general to test for presence, you can use `head_object` (as it will work the same as `get_object` but doesn't actually fetch the body)